### ISSUE TITLE
Converting a Stroke to a Path

### ISSUE THREAD

**Issue by photopea** [paperjs/paper.js#1968]
Your library contains many useful bézier path operations, e.g. boolean operations (union of two vector paths, intrsection, etc).

Would you be able to implement a stroke-to-path conversion? E.g. I have a square 100x100 pixels (four poins with lines between them), which needs to be stroked by a 8 pixel line. The result should be a clockwise square of 104x104 pixels, and a counter-clockwise circle of 96x96 pixels.

In other words, convert one vector path into another, such that stroking the input path "looks the same" as filling the output path.

Are there public algorithms for such task? What is this task called?

**Comment by eeropic** [comment]
https://github.com/paperjs/paper.js/issues/371

**Comment by eeropic** [comment]
Closing as duplicate of #371
